content/en/apps/guides/data/analytics/setup.md

```diff
@@ -0,0 +1,108 @@
+| `DBT_POSTGRES_USER`       | `postgres`                                         | Username of the PostgreSQL database where `DBT` creates tables and views from the models in `CHT_PIPELINE_BRANCH_URL`                          |
+| `DBT_POSTGRES_PASSWORD`   | `postgres`                                         | Password of the PostgreSQL database where `DBT` creates tables and views from the models in `CHT_PIPELINE_BRANCH_URL`                          |
+| `DBT_POSTGRES_SCHEMA`     | `dbt`                                              | PostgreSQL schema where `DBT` creates tables and views from the models in `CHT_PIPELINE_BRANCH_URL`                                            |
+| `DBT_POSTGRES_HOST`       | `postgres`                                         | PostgreSQL instance IP or endpoint                                                                                                             |
+| `CHT_PIPELINE_BRANCH_URL` | `"https://github.com/medic/cht-pipeline.git#main"` |                                                                                                                                                |
+| `COUCHDB_USER`            | `medic`                                            | Username of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_PASSWORD`        | `password`                                         | Password of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_DBS`             | `"medic"`                                          | Space separated list of databases to sync e.g `"medic medic_sentinel"`                                                                         |
+| `COUCHDB_HOST`            | `couchdb`                                          | Host of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_PORT`            | `5984`                                             | Port of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_SECURE`          | `false`                                            | Is connection to CouchDB instance secure?                                                                                                      |
+
+{{% alert title="Note" %}}
+If `CHT_PIPELINE_BRANCH_URL` is pointing to a private repo then you need to provide an access token in the url i.e. `https://<PAT>@github.com/medic/cht-pipeline.git#main`. In this example you will replace `<PAT>`  with an access token from Github. Instruction on how to generate one can be found [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+{{% /alert %}}
+
```

Comment:
We describe 3 setups here - awesome!  However, I wasn't actually told what to do when I was done. Please add some more content for this. Suggestions might be how to verify the command worked:

> You can verify this command worked by running `docker ps`.  It should show 6 containers running including Logstash, DBT, data generator, Postgres, CouchDB and Postgrest (note the `t` at the end!).  

As well, it's not clear you know how long to wait.  Maybe something like:

> The first time you run this it will need to download many docker images and will take a while.  You'll know it's done when you see `#8 DONE 0.0s` and your returned to the command line.  Be patient!

It wasn't actually clear to me what I'd achieved after running a command.  Suggest adding some next steps.  This might be:

> Now that all services are running, use a Postgres client like [Beekeeper](https://www.beekeeperstudio.io/) to connect to server `localhost:5432` with user `postgres` and password `postgres`.  You should be able to see sample data being inserted into the `medic_is_awesome` table.

Finally, we should mention how to clean things up for each scenario too. Maybe something like:

> When you're done using the services, you can clean everything by running`down`. Add `-v` at the to  remove all your data volumes: 
> 
> ```
> docker compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml down
> ```